src/features/projects/components/projectDetails/ProjectInfo.tsx

```diff
@@ -260,11 +279,74 @@ function ProjectInfo({ project }: Props): ReactElement {
                 })}
               </div>
             )}
+            {project?.metadata?.siteOwnerName ||
+            project?.metadata?.acquisitionYear ? (
+              <div className={styles.infoText}>
+                {project?.metadata?.siteOwnerName}{' '}
+                {project?.metadata?.siteOwnerName &&
+                project?.metadata?.acquisitionYear ? (
+                  'since '
+                ) : (
+                  <></>
+                )}
+                {!project?.metadata?.siteOwnerName &&
+                project?.metadata?.acquisitionYear ? (
+                  'Since '
+                ) : (
+                  <></>
+                )}
+                {project?.metadata?.acquisitionYear}
+              </div>
+            ) : (
+              <></>
+            )}
+          </div>
+        )}
+      </div>
 
-            <div className={styles.infoText}>
-              {project.metadata.siteOwnerName} since{' '}
-              {project?.metadata?.acquisitionYear}
+      <div style={{ display: 'flex' }}>
+        {project?.metadata?.landOwnershipType && (
+          <div className={styles.projectMoreInfo}>
+            <div className={styles.infoTitle}>
+              {t('manageProjects:siteOwnership')}
             </div>
+            {project?.metadata?.landOwnershipType && (
+              <div className={styles.infoText}>
+                {ownerTypes.map((ownerType: any, index: any) => {
+                  return (
+                    <React.Fragment key={ownerType}>
+                      {t(`manageProjects:${ownerType}`)}
+                      {index === ownerTypes.length - 2
+                        ? ' and '
```

Comment:
This should be translated!

Reply:
Actually ' and ' is occurring four times in this file. All places should use a translation resource instead.